### PR TITLE
build: dockerization

### PR DESCRIPTION
**Pull request by youngjoon-lee**
Dockerization for easy testing and deployment.
Please read the modified README first.

### CODE REVIEW

File: docs/usage-init-run.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+```
+By default, the app dir is generated as `$HOME/.doracle` in the enclave.
+It means that you can also find the generated app dir from your host (e.g. `/doracle/.doracle` or `$(pwd)/doracle/.doracle`).
+
+## Generate an oracle key
+
+NOTE: This step must be executed only by the first (genesis) oracle.
+
+```bash
+$DOCKER_CMD ego run doracled gen-oracle-key
```
Comment (0xHansLee) [inline docs/usage-init-run.md]: Trusted block info for light client seems to be required when `gen-oracle-key`.

**Review by 0xHansLee — APPROVED** [review]
Tested running doracle container.
Thanks!

**Review by gyuguen — APPROVED** [review]
LGTM

File: docs/usage-init-run.md (hunk 1 above)
Comment (youngjoon-lee) [reply docs/usage-init-run.md]: Done: [fd5b589](https://github.com/medibloc/panacea-doracle/pull/87/commits/fd5b589fbca6f708da272216f394e3dcd66331e8) & [58c78db](https://github.com/medibloc/panacea-doracle/pull/87/commits/58c78dbd10ff84f618848ebcbb33e100c8ff93d4)

**Pull request merged**